nebula/ui/components/VPNRecentConnections.qml

```diff
@@ -80,13 +80,22 @@ ColumnLayout {
                         let serverCityName = servers[x].slice(0, index).trim();
 
                         connection.push({
-                             countryCode: countryCode,
-                             serverCityName: serverCityName,
-                             localizedCityName: VPNLocalizer.localizedCityName(countryCode, serverCityName)
-                         });
+                            countryCode: countryCode,
+                            serverCityName: serverCityName,
+                            localizedCityName: VPNLocalizer.localizedCityName(countryCode, serverCityName)
+                        });
                     }
 
-                    append({isMultiHop, connection });
+                    const [{ firstCityLocalizedName: localizedCityName }, secondServer] = connection;
+                    const accessibleLabel = secondServer
+                        ? `${firstCityLocalizedName} ${VPNl18n.MultiHopFeatureMultiHopArrowAltText} ${secondServer.localizedCityName}`
```

Comment:
@flodolo what do you think about this string? We can also have a "$1 $2" in strings.yaml and let the translator decide what is better.

Reply:
Sorry, I don't understand what the ask is: are you suggesting to make this label localizable, e.g. `$1 to $2" or something else?

Reply:
Happy to make the string `$1 to $2` if we run the risk that this does not translate correctly in all cases.

Reply:
Let's make that, thanks 👍🏼

Reply:
Done!